docs/using/databricks.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: default
+title: Databricks
+description: Interact with your lakeFS data from Databricks
+parent: Using lakeFS with...
+nav_order: 60
+has_children: false
+---
+
+# Using lakeFS with Databricks
```

Comment:
In light of recent discoveries with the multicluster commiter, perhaps we need to document this here.